sortinghat/core/db.py

```diff
@@ -378,6 +427,71 @@ def to_none_if_empty(x):
     return uidentity
 
 
+def add_enrollment(uidentity, organization,
+                   start=MIN_PERIOD_DATE, end=MAX_PERIOD_DATE):
+    """Enroll a unique identity to an organization in the database.
+
+    The function adds a new relationship between the unique
+    identity in `uidentity` and the given `organization` to
+    the database.
+
+    The period of the enrollment can be given with the parameters
+    `from_date` and `to_date`, where `from_date <= to_date`.
+    Default values for these dates are `MIN_PERIOD_DATE` and
+    `MAX_PERIOD_DATE`. These dates cannot be `None`.
+
+    This function returns as result a new `Enrollment` object.
+
+    :param uidentity: unique identity to enroll
+    :param organization: organization where the unique identity is enrolled
+    :param start: date when the enrollment starts
+    :param end: date when the enrollment ends
+
+    :returns: a new enrollment
+
+    :raises ValeError: when either `start` or `end` are `None`;
```

Comment:
typo -> ValeError

Reply:
It should be fixed with the latest changes.